include/libcyphal/transport/can/delegate.hpp

```diff
@@ -230,26 +230,28 @@ class TransportDelegate
 
 };  // TransportDelegate
 
-/// This internal session delegate class serves the following purpose: it provides interface
-/// to access a session from transport (by casting canard's `user_reference` member to this class).
+// MARK: -
+
+/// This internal session delegate class serves the following purpose: it provides an interface (aka gateway)
+/// to access RX session from transport (by casting canard's `user_reference` member to this class).
 ///
-class SessionDelegate
+class IRxSessionDelegate
```

Comment:
Adding `I` prefix for anything like an interface (regardless whether it's internal or public stuff) - as it was requested in previous PR.